portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -279,14 +279,50 @@ definition {
 	}
 
 	@description = "Verify that a Numeric Field can be previewed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test NumericFieldCanBePreviewed {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133788 NumericFieldCanBePreviewed pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Numeric");
+
+		FormFields.editHelpText(
+			fieldHelpText = "Test Help Text",
+			fieldName = "tip");
+
+		FormFields.setPlaceholder(fieldValue = "Help Placeholder Test");
```

Comment:
@Diogomes, shouldn't the `fieldValue` be `Test Placeholder Text`?

Please check the Jira ticket